src/main/java/dev/openfeature/sdk/FeatureProviderWrapper.java

```diff
@@ -0,0 +1,118 @@
+package dev.openfeature.sdk;
+
+import dev.openfeature.sdk.exceptions.OpenFeatureError;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class FeatureProviderWrapper implements FeatureProvider, EventProviderListener {
```

Comment:
```suggestion
class FeatureProviderWrapper implements FeatureProvider, EventProviderListener {
```
I guess this doesn't need to be public?